test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs

```diff
@@ -44,32 +42,215 @@ public void ToOtlpLogRecordTest()
+            Assert.NotNull(Activity.Current);
+
+            var expectedTraceId = currentActivity.TraceId;
+            var expectedSpanId = currentActivity.SpanId;
+            Assert.Equal(expectedTraceId.ToString(), ActivityTraceId.CreateFromBytes(otlpLogRecord.TraceId.ToByteArray()).ToString());
+            Assert.Equal(expectedSpanId.ToString(), ActivitySpanId.CreateFromBytes(otlpLogRecord.SpanId.ToByteArray()).ToString());
+            Assert.Equal((uint)logRecord.TraceFlags, otlpLogRecord.Flags);
+        }
+
+        [Fact]
+        public void CheckToOtlpLogRecordSeverityText()
+        {
+            List<LogRecord> logRecords = new List<LogRecord>();
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.AddInMemoryExporter(logRecords);
+                    options.IncludeFormattedMessage = true;
+                });
+            });
+
+            var logger = loggerFactory.CreateLogger("OtlpLogExporterTests");
+            logger.LogInformation("Hello from {name} {price}.", "tomato", 2.99);
+            Assert.Single(logRecords);
+
+            var logRecord = logRecords[0];
+            var otlpLogRecord = logRecord.ToOtlpLog();
+
+            Assert.NotNull(otlpLogRecord);
+            Assert.Equal(logRecord.LogLevel.ToString(), otlpLogRecord.SeverityText);
+        }
+
+        [Fact]
+        public void CheckToOtlpLogRecordFormattedMessage()
+        {
+            List<LogRecord> logRecords = new List<LogRecord>();
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.AddInMemoryExporter(logRecords);
+                    options.IncludeFormattedMessage = true;
+                });
+            });
+
+            var logger = loggerFactory.CreateLogger("OtlpLogExporterTests");
+            logger.LogInformation("OpenTelemetry {Greeting} {Subject}!", "Hello", "World");
+            Assert.Single(logRecords);
+
+            var logRecord = logRecords[0];
+            var otlpLogRecord = logRecord.ToOtlpLog();
+
+            Assert.NotNull(otlpLogRecord);
+            Assert.Equal(logRecord.FormattedMessage, otlpLogRecord.Body.StringValue);
+        }
+
+        [Fact]
+        public void CheckToOtlpLogRecordExceptionAttributes()
+        {
+            List<LogRecord> logRecords = new List<LogRecord>();
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.AddInMemoryExporter(logRecords);
+                });
+            });
+
+            var exceptionMessage = "Exception Message";
+            var exception = new Exception(exceptionMessage);
+            var message = "Exception Occurred";
```

Comment:
Same here, consider removing the local variable.

Reply:
removed